test/test_config.py

```diff
@@ -145,6 +215,22 @@ def test_error_gs_dataset_mismatch(self):
         with pytest.raises(ValueError):
             config.init_global(test_config)
 
+    def test_config_values(self):
+        test_config = get_test_config()
```

Comment:
Are lines 219-220 needed if the function value_test_util is being used?

Reply:
This test does work without `get_test_config`, but I suspect that that is because proper global config cleanup isn't done properly in this entire test suite - `value_test_util` doesn't do any `get_test_config` setup.